token-metadata/js/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './errors.js';
+export * from './instruction.js';
+export * from './state.js';
```

Comment:
Shouldn't these be `*.ts`?

Reply:
Yes I agree its a bit weird, but i think its a quirk of the current prettier setup. I would usually not put the `.js/.ts` extension - but the prettier config forces it.  

If we do `.ts`, it raises the following typescript error:
`An import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.ts`

For what its worth its the same in this file:
https://github.com/solana-labs/solana-program-library/blob/5ce6eace7c128c3eb5686e02de983ee431b73ce1/token/js/src/index.ts

Reply:
Weird, but ok!